### DISCUSSION TITLE
Cüzdan Bağlama Sorunu

### DISCUSSION THREAD

**Discussion by onurdenizguvenilir** [IBAX-io/go-ibax#901] [Soru Cevap - Türkçe]
Merhabalar cüzdan bağlamada büyük sorun yaşıyorum bi türlü bağlayamadım acaba nasıl bağlaya bilirim resimli anlata bilecek birileri varmı acaba. 
teşekkürler...

**Comment by Yusufknk** [comment]
Cüzdanın nasıl bağlanacağı ile ilgili İbax anasayfasında redeem bölümünün altında bir video var onu izleyerek bağlayabilirsiniz. Private keylerinizi almayı unutmayın. Nasıl alınacağı ibax Türkiye grubunda mevcuttur.
